packages/components/src/tools-panel/tools-panel/hook.ts

```diff
@@ -48,6 +48,10 @@ const generateMenuItems = ( {
 	return menuItems;
 };
 
+const isMenuItemTypeEmpty = (
+	obj: ToolsPanelMenuItems[ ToolsPanelMenuItemKey ]
```

Comment:
If obj can be `undefined`, then this type should be updated to reflect that

```suggestion
	obj?: ToolsPanelMenuItems[ ToolsPanelMenuItemKey ]
```